test/yayson/presenter.coffee

```diff
@@ -282,6 +282,16 @@ describe 'Presenter', ->
     expect(json.data.relationships.car.links.related).to.eq '/cars/3/car'
     expect(json.data.relationships.car.data).to.eq undefined
 
+  it 'should not include relationships with nothing in them', ->
+    class CarPresenter extends Presenter
+      type: 'cars'
+
+      relationships: ->
+        car: CarPresenter
+
+    json = CarPresenter.render(id: 3)
+    expect(json.data.relationships.car).to.eq undefined
```

Comment:
I guess it's a matter of taste. I'd rather pretend the relationship didn't exist if we didn't specify it for a specific object. :)